components/infra-proxy-service/server/servers.go

```diff
@@ -40,7 +38,7 @@ func (s *Server) CreateServer(ctx context.Context, req *request.CreateServer) (*
 		return nil, status.Error(codes.InvalidArgument, "must supply server IP address")
 	}
 
-	server, err := s.service.Storage.StoreServer(ctx, req.Name, req.Description, req.Fqdn, req.IpAddress)
+	server, err := s.service.Storage.StoreServer(ctx, req.Id, req.Name, req.Fqdn, req.IpAddress)
```

Comment:
👍 Nice to see the (Name, Description) => (Id, Name) for cross-service consistency.